Sources/Sentry/SentrySpan.m

```diff
@@ -101,6 +124,12 @@ - (NSDictionary *)serialize
         }
     }
 
+    if (_tags != nil) {
+        @synchronized(_tags) {
+            [mutableDictionary setValue:_tags.copy forKey:@"tags"];
```

Comment:
`h`: As the span context can already contain tags we would override all of them here. Instead, we should merge them. If a tag is there twice the tag from SentrySpan wins.